x-pack/filebeat/cmd/otel.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.opentelemetry.io/collector/otelcol"
+)
+
+func OtelCmd() *cobra.Command {
+	command := &cobra.Command{
```

Comment:
Mark this command as hidden so that it does show up in the output of the `--help` option.

Also, explicitly log that it is experimental and unsupported.